antsibull-docs.cfg

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# Determines whether to add breadcrumbs to plugin docs via hidden sphinx toctrees. This can
+# take up a significant amount of memory if there are a large number of plugins so you can
+# disable this if necessary.
+breadcrumbs = true
+
+# Which docs parsing backend to use. Currently there is only 'ansible-core-2.13', and 'auto'
+# always chooses that one.
+doc_parsing_backend = "auto"
+
+# Determines whether to create the collection index and plugin indexes. They may not be
+# needed if you have a different structure for your website.
+indexes = true
+
+# Determines whether to use HTML blobs for option and return value tables. Using HTML blobs
+# reduces memory and CPU time usage significantly so you can disable this if necessary.
```

Comment:
Should this be

```suggestion
# reduces memory and CPU time usage significantly so you can enable this if necessary.
```

?

Reply:
Good spot. I copied this from the CLI argument help, so I also fixed that one in 8f6f4038f460dbcdb14168e3f018f8003a167d9c.